ArduPlane/Attitude.cpp

```diff
@@ -219,7 +219,7 @@ float Plane::stabilize_pitch_get_pitch_out()
        - throttle stick at zero thrust
        - in fixed wing non auto-throttle mode
     */
-    if (!quadplane_in_transition &&
+    if (!quadplane_in_frwd_transition &&
```

Comment:
this is specifically aiming for TVBS landing, so not fwd transition

Reply:
we will need @Hwurzburg to have a look at how this is defined, and for what classes of aircraft

Reply:
> tilt rotor VTOLs landing conventionally as fixed wing....with switch active in pilot throttle controlled modes it will force tilts up at low throttle to clear props...at highest switch position it also sets pitch to flare pitch .....use it on all my tvbs for emergency fw landing when batts are low

@Hwurzburg the reason we ask is that we want to decide if the `quadplane.in_transition()` test needs to be replaced with a check for forward transition only.
Based on your description, I understand now that the `!quadplane_in_transition` check is meant to protect against the case where the pilot is currently commanding a forward transition while on low throttle. In this case we don't want to force the landing pitch.

@tridge, on second inspection, I see that there is an additional check here for `!control_mode->is_vtol_mode()`. When `quadplane.is_back_transition()==true` then tautologically `is_vtol_mode()==true` as well.
So adding a definition of and using `is_back_transition()` is superfluous.

I think we can simply replace `in_transition()` with `in_frwd_transition()` and spare us the definition of `in_back_transition()`.

Reply:
I just tested the feature. I loaded Convergence in RealFligth, and set `long DO_AUX_FUNCTION 89 1`. This is the flare switch.
Took off in QSTAB, then I transitioned to FBWA.
Every time I closed the throttle to zero, the front tilt servos would raise the motors vertically.

Reply:
great!